tool/override-rolldown.js

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env node
+
+/**
+ * Rolldown Version Override Tool
+ * 
+ * This tool allows testing the vibe-dashboard with different rolldown-vite versions:
+ * - Last 5 stable versions from npm
+ * - Future/experimental versions from pkg.pr.new
+ */
+
+const { execSync } = require('child_process');
+const { readFileSync, writeFileSync } = require('fs');
+const { join } = require('path');
+
+const DASHBOARD_PACKAGE_PATH = join(process.cwd(), 'apps/dashboard/package.json');
+
+// Last 5 stable versions from npm (as of current check)
+const STABLE_VERSIONS = [
+  '7.1.4',
+  '7.1.3', 
+  '7.1.2',
+  '7.1.1',
+  '7.1.0'
+];
+
+// Future versions from pkg.pr.new (examples - these would be actual PR URLs)
+const FUTURE_VERSIONS = [
```

Comment:
Fetch future versions dynamically from 
https://pkg.pr.new/api/repo/commits?owner=rolldown&repo=rolldown

use `authoredDate` after last npm version date for future versions.